sympy/polys/factortools.py

```diff
@@ -124,13 +124,37 @@ def dmp_trial_division(f, factors, u, K):
 
 
 def dup_zz_mignotte_bound(f, K):
-    """Mignotte bound for univariate polynomials in `K[x]`. """
-    a = dup_max_norm(f, K)
-    b = abs(dup_LC(f, K))
-    n = dup_degree(f)
+    """
+    The Knuth-Cohen variant of Mignotte bound for
+    univariate polynomials in `K[x]`.
 
-    return K.sqrt(K(n + 1))*2**n*a*b
+    References
+    ==========
+
+    ..[1] [Abbott2013]_
+
+    """
+    from sympy import binomial
+
+    d = dup_degree(f)
+    delta = _ceil( d / 2 )
```

Comment:
In SymPy, no space is added after `(` and before `)`. So this could be `_ceil(d/2)`.